modules/users/client/controllers/confirm-email.client.controller.js

```diff
@@ -5,6 +5,18 @@
     .module('users')
     .controller('ConfirmEmailController', ConfirmEmailController);
 
+  function getEmailFromToken(token) {
+    // old tokens have lenght 40 symbols
```

Comment:
This comment could have a link to this PR so it'll be easier to understand what "old tokens" means.